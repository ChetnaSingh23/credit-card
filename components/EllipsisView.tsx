import React from 'react';
import {
    View,
  } from 'react-native';
import { bottomSheetStyles } from '../containers/debit-card/debit.style';

export const EllipsisView =() => {
    return (
    <View style={{display:'flex', flexDirection:'row'}}>
      <View style={bottomSheetStyles.ellipis} />
      <View style={bottomSheetStyles.ellipis} />
      <View style={bottomSheetStyles.ellipis} />
      <View style={bottomSheetStyles.ellipis} />
    </View>)
  }