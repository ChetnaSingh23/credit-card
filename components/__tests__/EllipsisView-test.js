import * as React from 'react';
import renderer from 'react-test-renderer';

import { EllipsisView } from '../EllipsisView';

it(`Ellipsis View renders correctly`, () => {
  const tree = renderer.create(<EllipsisView>Snapshot test!</EllipsisView>).toJSON();

  expect(tree).toMatchSnapshot();
});
