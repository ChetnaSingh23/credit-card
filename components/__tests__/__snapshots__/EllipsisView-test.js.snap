// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ellipsis View renders correctly 1`] = `
<View
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "borderRadius": 3,
        "fontWeight": "bold",
        "height": 6,
        "marginLeft": "5%",
        "top": 6,
        "width": 6,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "borderRadius": 3,
        "fontWeight": "bold",
        "height": 6,
        "marginLeft": "5%",
        "top": 6,
        "width": 6,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "borderRadius": 3,
        "fontWeight": "bold",
        "height": 6,
        "marginLeft": "5%",
        "top": 6,
        "width": 6,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "borderRadius": 3,
        "fontWeight": "bold",
        "height": 6,
        "marginLeft": "5%",
        "top": 6,
        "width": 6,
      }
    }
  />
</View>
`;
